app/models/authenticator/httpd.rb

```diff
@@ -157,5 +167,9 @@ def user_attrs_from_external_directory_via_dbus_api_service(username)
 
       HttpdDBusApi.new.user_attrs(username, ATTRS_NEEDED)
     end
+
+    def user_data_collected?(request)
```

Comment:
Can this method have a more descriptive name?  Maybe something like `authenticate_with_oauth2?`

Reply:
looking at what the method checks for, it can just be `request_authenticated?`  since the X-REMOTE-USER is passed to us only after the apache auth module(s) have succeeded auth.

Reply:
I think it might be best to eliminate the method and simply check:

`if request.headers['X-REMOTE-USER'].present?`